webapp/src/lib/parcelUtils.js

```diff
@@ -121,3 +121,50 @@ export function getParcelAttributes(wallet, parcel, districts) {
     backgroundColor: COLORS.taken
   }
 }
+
+export function drawMarker(
```

Comment:
_To consider_

I think this might deserve to be on it's own file. Like:

```javascript
import { marker } from './marker'

marker.draw(/* (...) */)
```

What do you think?